plugins/version-control-systems/git/src/main/kotlin/Git.kt

```diff
@@ -63,7 +63,7 @@ import org.semver4j.RangesListFactory
 const val GIT_HISTORY_DEPTH = 50
 
 // Replace prefixes of Git submodule repository URLs.
-private val REPOSITORY_URL_PREFIX_REPLACEMENTS = mapOf(
+private val REPOSITORY_URL_PREFIX_REPLACEMENTS = listOf(
```

Comment:
From the commit message I do not understand whether there was an issue with the previous approach. 
Did it not work in some cases?

Reply:
The previous approach only did the URL replacement on the call of `getNested()`, whereas the underlying repository configuration (for Git, inside `.git`) still contained the original URLs. Which means that any incremental update to that working tree would have failed.